tests: mimxrt11xx series build failure

**Describe the bug**

```

CASE_NAME	ZEPHYR_VERSION	RESULT	PLATFORM	TEST_PATH
kernel.tickless.concept	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/kernel/tickless/tickless_concept
kernel.common.profiling	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/kernel/profiling/profiling_api
kernel.timer.tickless	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/kernel/timer/timer_api
pm.device_runtime.api	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/subsys/pm/device_runtime_api
pm.device_runtime.isr_safe.api	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/subsys/pm/device_runtime_api
benchmark.kernel.footprints.pm	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1160_evk/mimxrt1166/cm4	tests/benchmarks/footprints
benchmark.kernel.footprints.pm	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/benchmarks/footprints
pm.device_runtime.api	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/subsys/pm/device_runtime_api
pm.device_runtime.isr_safe.api	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/subsys/pm/device_runtime_api
kernel.timer.tickless	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/kernel/timer/timer_api
kernel.common.profiling	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/kernel/profiling/profiling_api
kernel.tickless.concept	v3.6.0-337-g72a17b6d887e	Build failure	mimxrt1170_evk@B/mimxrt1176/cm4	tests/kernel/tickless/tickless_concept
```

**Logs and console output**

```
map=/home/jenkins/agent/workspace/bitbucket_build_platform=WEST_TOPDIR -ffunction-sections -fdata-sections --specs=picolibc.specs -std=c99 -MD -MT zephyr/CMakeFiles/zephyr.dir/soc/nxp/imxrt/imxrt11xx/power.c.obj -MF zephyr/CMakeFiles/zephyr.dir/soc/nxp/imxrt/imxrt11xx/power.c.obj.d -o zephyr/CMakeFiles/zephyr.dir/soc/nxp/imxrt/imxrt11xx/power.c.obj -c /home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/soc/nxp/imxrt/imxrt11xx/power.c
/home/jenkins/agent/workspace/bitbucket_build_platform/zephyr/soc/nxp/imxrt/imxrt11xx/power.c:12:10: fatal error: power_rt11xx.h: No such file or directory
   12 | #include "power_rt11xx.h"
      |          ^~~~~~~~~~~~~~~~
compilation terminated.
ninja: build stopped: subcommand failed.
```

**Environment (please complete the following information):**
 - OS: (e.g. Linux, MacOS, Windows)
 - Toolchain (e.g Zephyr SDK, ...)
 - Commit SHA or Version used: v3.6.0-337-g72a17b6d887e

--- a/soc/nxp/imxrt/imxrt10xx/lpm_rt1064.c
+++ b/soc/nxp/imxrt/imxrt10xx/lpm_rt1064.c
@@ -214,6 +214,7 @@ void clock_full_power(void)
 #if DT_SAME_NODE(DT_NODELABEL(flexspi), DT_PARENT(DT_CHOSEN(flash)))
 	clock_set_div(kCLOCK_FlexspiDiv, flexspi_div);
 	clock_set_mux(kCLOCK_FlexspiMux, 3);
+#endif
 #if DT_SAME_NODE(DT_NODELABEL(flexspi2), DT_PARENT(DT_CHOSEN(flash)))
 	clock_set_div(kCLOCK_Flexspi2Div, flexspi_div);
 	clock_set_mux(kCLOCK_Flexspi2Mux, 1);
@@ -261,6 +262,7 @@ void clock_low_power(void)
 	clock_set_div(kCLOCK_FlexspiDiv, 0);
 	/* FLEXSPI1 mux to PLL3 PFD0 BYPASS */
 	clock_set_mux(kCLOCK_FlexspiMux, 3);
+#endif
 #if DT_SAME_NODE(DT_NODELABEL(flexspi2), DT_PARENT(DT_CHOSEN(flash)))
 	clock_set_div(kCLOCK_Flexspi2Div, 0);
 	/* FLEXSPI2 mux to PLL3 PFD0 BYPASS */

--- a/soc/nxp/imxrt/imxrt11xx/power.c
+++ b/soc/nxp/imxrt/imxrt11xx/power.c
@@ -9,7 +9,7 @@
 #include <fsl_dcdc.h>
 #include <fsl_gpc.h>
 #include <zephyr/dt-bindings/pm/imx_spc.h>
-#include "power_rt11xx.h"
+#include "power.h"
 
 #include <zephyr/logging/log.h>
 LOG_MODULE_DECLARE(soc, CONFIG_SOC_LOG_LEVEL);

